File: Archive-2015/5. Views-Graphics-in-iOS/LoadingNibFiles/LoadingNibFiles/ViewController.m
//
//  ViewController.m
//  LoadingNibFiles
//
//  Created by Doncho Minkov on 10/29/14.
//  Copyright (c) 2014 Telerik Academy. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

static NSString* viewIdentifier = @"DetailsView";

- (void)viewDidLoad {
    [super viewDidLoad];
    NSArray* nibs = [[NSBundle mainBundle] loadNibNamed:@"DetailsView"
                                                     owner:self
                                                options:nil];
    UIView *subview = (UIView*)[nibs objectAtIndex:0];
    
//    [self.view addSubview:subview];
    
    CGRect rect = CGRectMake(50, 50, 30, 75);
    UIButton *but = [[UIButton alloc] initWithFrame:rect];
//    but.backgroundColor = [UIColor blackColor];
//    but.titleLabel.font = [UIFont boldSystemFontOfSize:28];
//    but.titleLabel.textColor = [UIColor yellowColor];
//    but.titleLabel.text = @"Click";
    
    [self.view addSubview:but];

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Resources/Archive-2016/demos-news/AnimationsDemo/AnimationsDemo/ViewController.swift
//
//  ViewController.swift
//  AnimationsDemo
//
//  Created by Doncho Minkov on 2/2/16.
//  Copyright © 2016 Doncho Minkov. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var _view: UIView?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let parentViewFrame = self.view.frame
        
        let frame = CGRectMake(parentViewFrame.midX - 75,
                    parentViewFrame.midY - 75,
                    150, 150);
        
        _view = AnimatedView(frame: frame);
        
        _view?.backgroundColor = UIColor.blackColor()
        
        self.view.addSubview(_view!)
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
    
    
    override func touchesBegan(touches: Set<UITouch>, withEvent event: UIEvent?) {
        _view?.touchesBegan(touches, withEvent: event)
    }
    
    
}

